util/a2e/atoe_utils.c

```diff
@@ -115,7 +115,7 @@ pchar(InstanceData *this, int c)
  *               str            String to add to buffer
  *               left_justify   Left justify string flag
  *               min_width      Minimum width of string added to buffer
- *               precision
+ *               precision      Maximum chars to take from the argument
  * returns     - int return code, 0 for success
  *************************************************************************/
 static int
```

Comment:
Suggest the second parameter should be `const char *str`.
While you're here, please ensure locals have initializers where reasonable; e.g. line 125 should be
```c
	int pad_length = 0;
```